src/OracleService/Protocols/OracleNeoFSProtocol.cs

```diff
@@ -76,36 +81,37 @@ private Task<byte[]> GetAsync(Uri uri, string host, CancellationToken cancellati
             };
         }
 
-        private static async Task<byte[]> GetPayloadAsync(Client client, Address addr, CancellationToken cancellation)
+        private static async Task<string> GetPayloadAsync(Client client, Address addr, CancellationToken cancellation)
         {
             Object obj = await client.GetObject(cancellation, new GetObjectParams() { Address = addr }, new CallOptions { Ttl = 2 });
-            return obj.Payload.ToByteArray();
+            return Convert.ToBase64String(obj.Payload.ToByteArray());
```

Comment:
I think it should be `Utility.StrictUTF8.GetString(obj.Payload.ToByteArray())`. Because we support JSON only for now, right?

Reply:
https://github.com/neo-project/neo-modules/blob/603ecec356d4ec675023501cb59c24d1e4ded73b/src/OracleService/OracleService/config.json#L7

Reply:
And is there is way to check the content-types of the neofs object?

Reply:
The only way I see is to define an optional well-known [`Attribute`](https://github.com/nspcc-dev/neofs-api/blob/master/object/types.proto#L137) for content-type and check it, if it was set.

Reply:
Done.